Editor/Localization/ja.po

```diff
@@ -108,19 +120,19 @@ msgid "Slack Mention Settings"
 msgstr "Slackメンション設定"
 
 # mentionSubTeamIDs
-msgid "Mention Sub Team IDs"
+msgid "Sub Team IDs to Mention"
```

Comment:
Comma-separated value is accepted. This is mentioned by the tooltip of this input. The input name should still be short because it should be readable if the display area is very tiny. So I mentioned about it by the tooltip, not the name.

![Screenshot 2023-11-15 at 9 04 59](https://github.com/DeNA/Anjin/assets/1124024/18f7cf84-1d33-4f09-a145-a5994051cf2b)

Reply:
I understanded. LGTM